### PR TITLE
Fixing minor documentation errors

### PR DESCRIPTION
**Pull request by richardkapolnai-da**

### CODE REVIEW

File: docs/source/daml/intro/1_Token.rst [hunk 1]
```diff
@@ -34,17 +34,17 @@ DAML specifies what transactions are legal on a DAML Ledger. The rules the DAML
...
-  :lines: 6, 9
+  :lines: 5, 9
 
-Code comments in DAML are introduced with `--`:
+Code comments in DAML are introduced with ``--``:
 
 .. literalinclude:: daml/daml-intro-1/Token.daml
   :language: daml
-  :lines: 4-9
+  :lines: 1-5
```
Comment (richardkapolnai-da) [inline docs/source/daml/intro/1_Token.rst:47]: I changed this to show the copyright message to keep the size:
```
-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
-- SPDX-License-Identifier: Apache-2.0

-- A DAML file defines a module.
module Token where
```

Cf [current version](https://docs.daml.com/daml/intro/1_Token.html)

File: docs/source/daml/intro/1_Token.rst [hunk 2]
```diff
@@ -34,17 +34,17 @@ DAML specifies what transactions are legal on a DAML Ledger. The rules the DAML
 DAML files and modules
 ----------------------
 
-Each ``.daml`` file defines a *DAML Module*. At the top of each DAML file is a pragma informing the compiler of the language version and the module name:
+Each ``.daml`` file defines a *DAML Module* at the top:
 
 .. literalinclude:: daml/daml-intro-1/Token.daml
   :language: daml
-  :lines: 6, 9
+  :lines: 5, 9
```
Comment (richardkapolnai-da) [inline docs/source/daml/intro/1_Token.rst:41]: I manually verified because no idea what `9` means here

File: docs/source/daml/intro/1_Token.rst [hunk 3]
```diff
@@ -53,7 +53,7 @@ A ``template`` defines a type of contract that can be created, and who has the r
 
 .. literalinclude:: daml/daml-intro-1/Token.daml
   :language: daml
-  :lines: 16,21,25,29,34
+  :lines: 12,17,21,25,30
```
Comment (richardkapolnai-da) [inline docs/source/daml/intro/1_Token.rst:56]: I manually verified too

File: docs/source/daml/intro/1_Token.rst (hunk 2 above)
Comment (bame-da) [reply docs/source/daml/intro/1_Token.rst:41]: It's a list of lines. This means "show lines 5 and 9".

File: docs/source/daml/intro/1_Token.rst (hunk 1 above)
Comment (bame-da) [reply docs/source/daml/intro/1_Token.rst:47]: I'd just do 4-5.

File: docs/source/daml/intro/daml/daml-intro-4/Contact.daml [hunk 4]
```diff
@@ -49,7 +49,7 @@ choice_test = do
       address = "1 Bobstreet"
       telephone = "012 345 6789"
 
-  -- The bank can't change its own telephone number as the accountant controls
+  -- Bob can't change its own telephone number as Alice controls
```
Comment (bame-da) [inline docs/source/daml/intro/daml/daml-intro-4/Contact.daml:52]:
```suggestion
  -- Bob can't change his own telephone number as Alice controls
```

**Review by bame-da — APPROVED** [review]

File: docs/source/daml/intro/1_Token.rst (hunk 2 above)
Comment (richardkapolnai-da) [reply docs/source/daml/intro/1_Token.rst:41]: oh. then `6` was a mistake, so now `9` is a mistake, both were/are empty lines ignored by doc builder :)
so changing to 
```suggestion
  :lines: 5
```

**Pull request merged**